src/main/java/de/tum/in/www1/artemis/domain/MathSubmission.java

```diff
@@ -0,0 +1,93 @@
+package de.tum.in.www1.artemis.domain;
+
+import static de.tum.in.www1.artemis.config.Constants.MAX_SUBMISSION_TEXT_LENGTH;
+
+import javax.persistence.*;
+import javax.validation.constraints.Size;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+import de.tum.in.www1.artemis.domain.enumeration.Language;
+
+/**
+ * A MathSubmission.
+ */
+@Entity
+@DiscriminatorValue(value = "MATH")
+@SecondaryTable(name = "math_submission_details")
+@JsonInclude(JsonInclude.Include.NON_EMPTY)
+public class MathSubmission extends Submission {
+
+    @Override
+    public String getSubmissionExerciseType() {
+        return "math";
+    }
+
+    private static final int MAX_EXCERPT_LENGTH = 100;
+
+    @Column(name = "text", table = "math_submission_details")
+    @Size(max = MAX_SUBMISSION_TEXT_LENGTH, message = "The text submission is too large.")
+    private String text;
+
+    @Enumerated(EnumType.STRING)
+    @Column(name = "language", table = "math_submission_details")
+    private Language language;
```

Comment:
This column already exists as part of the `submission` base table, so you should reuse that one, instead of creating your own.

Reply:
yea don't know if I'll use / need this at all, removed it in general